network/ios/ios_facts.py

```diff
@@ -118,9 +118,14 @@
 import re
 
 from ansible.module_utils.basic import get_exception
-from ansible.module_utils.netcmd import CommandRunner
+from ansible.module_utils.netcli import CommandRunner, AddCommandError
 from ansible.module_utils.ios import NetworkModule
 
+def add_command(runner, command):
```

Comment:
Could you please use two blank lines before `def` will make it easier to enforce pep8ish later on.